openvpn/testdata/100-milisec-process.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+echo "Starting"
+sleep 0.2
```

Comment:
why is `100-milisec-process` doing 200ms sleep? :D

Reply:
Nice catch :) fixed